Resources/Audio/DeltaV/Jukebox/attributions.yml

```diff
@@ -0,0 +1,77 @@
+# sorted alphabetically based on filenames in the folder Resources/Audio/DeltaV/Jukebox
+# keep it ordered or I'll stab you
+
+- files: ["a_different_reality_lagoona_remix.xm-MONO.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "A.D.R (Lagoona rmx) by Andreas Viklund, converted to mono"
+  source: "https://modarchive.org/index.php?request=view_by_moduleid&query=134786"
+
+- files: ["aggravated.it-MONO.ogg"]
+  license: "CC-BY-NC-SA-4.0"
+  copyright: "MEL -Aggravated Assault by melcom, converted to mono"
+  source: "https://modarchive.org/index.php?request=view_by_moduleid&query=176234"
+
+- files: ["autumnal_equinox.xm-MONO.ogg"]
+  license: "CC-BY-NC-4.0"
+  copyright: "Autumnal Equinox by lemonade, converted to mono"
+  source: "https://modarchive.org/index.php?request=view_by_moduleid&query=143993"
+
+- files: ["DOS=HIGH,_UMB.ogg"]
+  license: "Custom"
```

Comment:
can we get "custom" defined so it's recorded somewhere what the license is?

Reply:
I have express permission from MBR to use this track. It says this in the copyright line immediately below this.

Reply:
read it after but already sent the comment :(